src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -102,6 +105,9 @@ public function testHas()
         $this->assertTrue($builder->has('bar'), '->has() returns true if a service exists');
     }
 
+    /**
+     * @group legacy
```

Comment:
not all this test is legacy. It would be a great time to split the test into multiple smaller tests to mark only the relevant one as legacy